### PR TITLE
Court dropdown

### PR DESCRIPTION
**Pull request by AlexRiina**
I don't have a list of courts to pull from, so this has a bit less structure than facilities. They're both kind of strange fields and could be rewritten as ids into the courts and facilities tables.

### CODE REVIEW

**Comment by AlexRiina** [comment]
Partial implementation of #54

**Comment by AlexRiina** [comment]
woops forgot to add migration and factories in first push (updated now)

File: db/seeds.rb
Comment (RSid) [inline db/seeds.rb]: Do you want to reach out to Atara and ask her for a list of these? You could also generate a starting list by looking at the data we currently have in the court field on clients and getting a list of distinct entries.

It probably won't be complete right off the bat, but we'd want it to have as much data as possible before merging this since we'd be going from a free text field to a dropdown

File: app/views/admin/clients/_edit_case_modal.html.erb
Comment (RSid) [inline app/views/admin/clients/_edit_case_modal.html.erb]: We're using the simple_form library for our forms, & this doesn't show up in the UI for me - take a look at the facilities field as an example of how to do a select fields with a collection like this. 

In particular, if you pull master you'll see an example of how to make the selected value be the name field instead of the default id

Comment (AlexRiina) [reply app/views/admin/clients/_edit_case_modal.html.erb]: The datalist tag should be hidden, but create a dropdown of suggestions for filling in the text field. 

I think it works in most browsers but not safari apparently. Here's what it looks like for me (with an empty text input)
court-dropdown-new
court-dropdown-1 and here's a live example from w3schools https://www.w3schools.com/tags/tryit.asp?filename=tryhtml5_datalist.

My original attempt was going to be making it free text while we figured out what the options should be, but I'll see if I can get a complete list from Atara and make this a little stricter.

**Comment by AlexRiina** [comment]
Just reimplemented it as a dropdown with the list from Atara. I'm not sure
the database is 100% according to the list, so the edit form will display a
message if the value doesn't match a known value and will also convert a
string like "Ayer" into the number of the row when saved. It's not great,
but it's a starting point to making it a foreign key I think.

File: app/views/admin/clients/_edit_case_modal.html.erb
Comment (RSid) [reply app/views/admin/clients/_edit_case_modal.html.erb]: Ahh, interesting, nice! I'd say this is fine as-is as well, as long as we're sure to explain to Atara what to expect.

**Comment by RSid** [comment]
@AlexRiina sorry for the mergeconflicts - I think it will save us some hassle if we do our developing right now against the legacy client referral form (the old weebly/PHP one we talked about on our call), since for now that matches production. 

I've moved the simpleform version of the client referral form into its own file for later use, and pulled in the legacy form in its place. Let me know if you run into any issues with it!

**Review by RSid — APPROVED** [review]
Looks good! Approved pending mergeconflict resolution.

**Pull request merged**
